Tests/SynchronizableTests.swift

```diff
@@ -23,12 +23,28 @@ import Foundation
 import XCTest
 
 
+extension Valet {
+
+    // MARK: Accessibility -> CloudAccessibility conversion
+
+    var cloudAccessibility: CloudAccessibility {
+        switch flavor {
+        case .iCloud(let cloudAccessibility):
+            return cloudAccessibility
+        case .vanilla:
+            assert(false)
+            return .always
+        }
+    }
+}
+
+
 @available (iOS 8.2, OSX 10.11, *)
```

Comment:
I don't think we need this `@available` anymore

Reply:
What are our target minimums now? A few of these may be out of date

Reply:
iOS 9.0, and macOS 10.11